app/lib/html_css_to_image.rb

```diff
@@ -14,13 +14,13 @@ def self.url(html:, css: nil, google_fonts: nil)
 
   def self.fetch_url(html:, css: nil, google_fonts: nil)
     cache_key = "htmlcssimage/#{html}/#{css}/#{google_fonts}"
-    cached_url = Rails.cache.read(cache_key)
+    cached_url = RedisClient.get(cache_key)
```

Comment:
@benhalpern pinging you here because I had an idea on how to avoid the "newly cold cache" scenario. We could also warm up Redis in the background before/after this PR, while keeping the worst case scenario of the double read in

Reply:
I think there may be some circumstances where we want to do this but I would rather first go with moving things over cold turkey. My reasons for this are:
-  Memcache has keys in it that don't expire, I would worry that some keys might never stop being pulled from Memcache and that it would be nearly impossible to tell what we were still pulling from Memcache and what was now being pulled from Redis.
- "Technically" we should be able to handle blowing away our caches(though I fully get this is usually not the case!) and those caches that we can't handle blowing away, I want to know what those are so that I can help design a more robust system to prevent those caches from missing in the future.
- Rolling over cold turkey feels more simple and straightforward to me and clear about what is in Redis and what isnt. I think that clarity actually reduces risk in a way and in the process will go a long way in helping us understand what and how we use caching

Reply:
I think _the vast majority_ of our cases will be fine with cold turkey rollover. We just started with a tough one. I'm also in favor of finding all the low hanging fruit in with simple turnover and dealing with possibly more difficult scenarios later.